test/connect.spec.js

```diff
@@ -10,6 +10,7 @@ import Vuex from 'vuex'
 describe('connect', () => {
   const TEST = 'TEST'
   let state, getters, actions, mutations, store, options, Component
+  let namespaced_state, namespaced_getters, namespaced_store
```

Comment:
Please prefer lowerCamelCase. The same for all further appearance of variables/properties.